neuror/cut_plane/cut_leaves.py

```diff
@@ -30,6 +30,9 @@ def _get_cut_leaves(half_space, morphology, bin_width, percentile_threshold):
 
     # compute the min cut leave given the percentile
     projected_uncut_leaves = half_space.project_on_directed_normal(leaves_coord[~cut_filter])
+    if not projected_uncut_leaves.tolist():
```

Comment:
Can you to use `projected_uncut_leaves.size == 0` for explicity

Reply:
ah good, I didn't know this .size, thanks!